bionemo-recipes/recipes/llama3_native_te/README.md

```
@@ -100,6 +100,27 @@ configuration parameters, including switching to `MXFP8BlockScaling`, can be set
+
+```python
+python train_fsdp2.py \
+fp8_stats_config.enabled=True # whether to log stats or not
+fp8_stats_config.fp8_log_dir=./logs/fp8_stats_logs_dummy # where to store the logs
+fp8_stats_config.fp8_stats_file=./fp8_debugging_stats.yaml # specifies what stats you want to run. Currently this is saved in this yaml file.
+fp8_config.enabled=True # set this to use FP8 otherwise stats logging wont work
+```
+
+Note: This feature is available for the `train_ddp` and the `train_fsdp2` scripts. It is not yet available for `train_mfsdp`.
+
+The config file structure [fp8_debugging_stats.yaml](fp8_debugging_stats.yaml) is explained [here](https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/debug/2_config_file_structure.html) in more detail. Below we will cover some very basic elements of the file structure.
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)